model_analyzer/config/generate/config_parameters.py

```diff
@@ -26,23 +28,23 @@ class ConfigParameters:
     Contains information about all configuration parameters the user wants to search
     """
 
-    def __init__(self):
-        self._parameters: Dict[str, ConfigParameter] = {}
+    # These map to the run-config-search fields (see config.md for details)
+    exponential_rcs_parameters = ["batch_sizes", "concurrency"]
```

Comment:
I think I'm still missing the context here. Could we sync offline about this? I think something isn't clear here, and I wonder if we could resolve it.

Reply:
Decision: Just add headers for the two "types". The comment with the explanation and documentation link will go into SearchParameter.